File: tests/fixtures/local-dev-samples-legacy/new-rust/handler.rs
use serde_json::{json, Value};

pub fn handler(event: Value) -> Value {
    json!({
        "status": 200,
        "headers": { "Content-Type": "application/json" },
        "body": json!({
            "message": "Hello from FastFn Rust!",
            "input": event
        }).to_string()
    })
}

File: tests/fixtures/local-dev-samples-migrated/usuarios-api/handler.js
/**
 * @typedef {Object} Context
 * @property {string} requestId
 * @property {Object} headers
 */

/**
 * Handle the request.
 * @param {Object} event - The input event
 * @param {Context} context - The execution context
 * @returns {Promise<Object>} The response
 */
module.exports.handler = async (event, context) => {
  return {
    status: 200,
    headers: {
      "Content-Type": "application/json"
    },
    body: JSON.stringify({
      message: "Hello from FastFN Node!",
      input: event
    })
  };
};
